src/components/language/ethnologue-language/ethnologue-language.service.ts

```diff
@@ -292,11 +284,12 @@ export class EthnologueLanguageService {
           response[record.property] = {};
         }
         if (record?.property) {
+          const canRead = perms[record.property]
+            ? perms[record.property].canRead
+            : false;
```

Comment:
FYI this could be simplified
```suggestion
          const canRead = perms[record.property]?.canRead ?? false;
```